urlshort/README.md

```diff
@@ -0,0 +1,36 @@
+# URL Shortener
+Exercise details from https://gophercises.com/exercises/urlshort
+
+The goal of this exercise is to create an [http.Handler](https://golang.org/pkg/net/http/#Handler) that will look at the path of any incoming web request and determine if it should redirect the user to a new page, much like URL shortener would.
+
+For instance, if we have a redirect setup for `/dogs` to `https://www.somesite.com/a-story-about-dogs` we would look for any incoming web requests with the path `/dogs` and redirect them.
+
+To complete this exercises you will need to implement the stubbed out methods in [handler.go](handler.go). There are a good bit of comments explaining what each method should do, and there is also a [main/main.go](main/main.go) source file that uses the package to help you test your code and get an idea of what your program should be doing.
+
+I suggest first commenting out all of the code in main.go related to the `YAMLHandler` function and focusing on implementing the `MapHandler` function first.
+
+Once you have that working, focus on parsing the YAML using the [gopkg.in/yaml.v2](https://godoc.org/gopkg.in/yaml.v2) package. *Note: You will need to `go get` this package if you don't have it already.*
+
+After you get the YAML parsing down, try to convert the data into a map and then use the MapHandler to finish the YAMLHandler implementation. Eg you might end up with some code like this:
+
+```go
+func YAMLHandler(yaml []byte, fallback http.Handler) (http.HandlerFunc, error) {
+  parsedYaml, err := parseYAML(yaml)
+  if err != nil {
+    return nil, err
+  }
+  pathMap := buildMap(parsedYaml)
+  return MapHandler(pathMap, fallback), nil
+}
+```
+
```

Comment:
Your README is a bit confusing. Does it contain the description of the code you wrote or the actual task you were supposed to do? In the sample code above there is go routine created but your code does not have an go routines implementation.

Reply:
the README is just a description of the exercise